gossip/hub/configpb/config.proto

```diff
@@ -0,0 +1,74 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package configpb;
+
+import "github.com/google/trillian/crypto/keyspb/keyspb.proto";
+import "github.com/google/trillian/crypto/sigpb/sigpb.proto";
+import "google/protobuf/any.proto";
+
+// HubMultiConfig wraps up a HubBackendSet and corresponding HubConfigSet so
+// that they can easily be parsed as a single proto.
+message HubMultiConfig {
```

Comment:
Should a multi-config be a repeated `HubConfig`?

Reply:
👍 for `HubMultiConfig`

Replacing `HubBackendSet` would only be worthwhile if it was replaced with a `map<string, string>` rather than `repeated string`

Reply:
`map<string,string>` it is.